app/workers/evss/document_upload.rb

```diff
@@ -1,11 +1,11 @@
 # frozen_string_literal: true
-class DisabilityClaim::DocumentUpload
+class EVSSClaim::DocumentUpload
```

Comment:
This change may impact any queued jobs when its deployed as the class name stored will be the old name. Not sure best way to handle it, a migration/task that removes it from the queue and queues it with this class? The number of jobs should be small or even zero, but still a risk. Another option would be to keep class with old name and pass through arguments to this class, or have it inherit from this class